src/Tribe/Main.php

```diff
@@ -487,7 +487,12 @@ protected function addHooks() {
 			add_action( 'plugins_loaded', array( 'Tribe__Cache', 'setup' ) );
 			add_action( 'plugins_loaded', array( 'Tribe__Support', 'getInstance' ) );
 			add_action( 'plugins_loaded', array( $this, 'set_meta_factory_global' ) );
-			add_action( 'current_screen', array( $this, 'init_admin_list_screen' ) );
+
+			if ( ! ( defined( 'DOING_AJAX' ) && DOING_AJAX ) ) {
```

Comment:
Maybe use `Tribe__Main::instance()->doing_ajax()` here and line 2333? (Possible testability benefit.)